sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/autorest.md

```diff
@@ -0,0 +1,108 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: SecurityInsights
+namespace: Azure.ResourceManager.SecurityInsights
+require: https://github.com/Azure/azure-rest-api-specs/blob/53b1affe357b3bfbb53721d0a2002382a046d3b0/specification/securityinsights/resource-manager/readme.md
+tag: package-preview-2022-08
+output-folder: $(this-folder)/Generated
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  flatten-payloads: false
+
+request-path-to-parent:
+  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/main/indicators: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/main/indicators/{name}
+
+request-path-to-resource-data:
+  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/main/indicators/{name}: ThreatIntelligenceIndicatorModel
+
+operation-positions:
+  ThreatIntelligenceIndicators_List: collection
+
+rename-mapping:
+  ThreatIntelligenceIndicatorModel: ThreatIntelligenceIndicator
+
+override-operation-name:
+  DomainWhois_Get: GetDomainWhoisInformation
```

Comment:
Should this be WhoIs instead of Whois? The spellcheck also flagged this

Reply:
The office site says `Whois` :)